bounce/server/api/clubs.py

```diff
@@ -89,11 +93,17 @@ class SearchClubsEndpoint(Endpoint):
         """Handles a GET /club/search request by returning
         clubs that contain content from the query."""
 
-        query = None
+        # default values, TODO: set default value in json-schema
+        query = ''
```

Comment:
I think `query` is supposed to default to `None` here. When we just want to display a few recently created clubs on the landing page we make this call with no query, and the `search` function in `db/club.py` sees that `query` is `None` and just returns the most recently created clubs.

Reply:
Hmm.. maybe! I will be more careful in the future....